indico/modules/oauth/controllers.py

```diff
@@ -313,6 +323,9 @@ class RHRevokePersonalToken(RHPersonalTokenBase):
     def _process(self):
         self.token.revoke()
         logger.info('Revoked token %r', self.token)
+        self.user.log(UserLogRealm.user, LogKind.negative, 'API Tokens', 'Token revoked', session.user,
+                        data={'ID': self.token.id, 'Name': self.token.name},
+                        meta={'token_id': self.token.id})
```

Comment:
```suggestion
                      data={'ID': self.token.id, 'Name': self.token.name},
                      meta={'token_id': self.token.id})
```

Reply:
god I HATE this stupid vscode behavior that [shift-]tab on block selection does not simply dedent/indent ALL lines by the same amount of spaces...